config/initializers/gem-plugin_config/sanitizer_config.rb

```diff
@@ -38,17 +39,26 @@ module Config
         'img' => {'src' => ['http', 'https', :relative]},
         'q' => {'cite' => ['http', 'https', :relative]}
       }
-    }
+    )
+
+    CLASS_ATTRIBUTE = freeze_config(
+      # see in the Transformers section for what classes we strip
+      attributes: {
+        all: ['align', 'title', 'dir', 'class'],
+      }
+    )
+
+    CSS_ALLOWED = merge(ARCHIVE, CLASS_ATTRIBUTE)
```

Comment:
If `CLASS_ATTRIBUTE` is not used elsewhere you can inline it here. `CSS_ALLOWED` needs a `freeze_config` on it.